content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,289 @@
+
+![Registering model versions](/img/mr-register-model-version.gif)
+
+Since we just saved our model to dvc and added it to the model registry in the
+previous commit, we can just keep the commit which was selected automatically.
+We will also keep the suggested version number. For more details and other ways
+of registering model versions you can have a look at the corresponding
+[documentation](/doc/studio/user-guide/model-registry/register-version).
+
+Once we register our first model version, the model registry will also
+automatically connect it to experiment tracking and all metrics which are
+tracked for each model version will also show up in the model registry. We can
+even explore the experiment directly by clicking on the "Open in Project" button
+on the model detail page.
+
+<details>
+
+#### 💡 Expand to see how registering models works under the hood
+
+Registering the model version as we just did using DVC Studio is equivalent to
+the following GTO command
+
+```
+gto register pool-segmentation [ref] --version v1.0.0
+```
+
+Here, `[ref]` is the git reference/hash we selected from the menu in Studio.
+
+For more details you can have a look at the
+[gto register command reference](doc/gto/command-reference/register).
+
+</details>
+
+## Assigning lifecycle stages
+
+We have a first version for our model and now it is a good time to assign a
+model lifecycle stage to it. You can create any number of lifecycle stages with
+any names you wish but in this example we will only create two stages called
+"dev" and "test".
+
+Stages are created whenever a model version is assigned to them. We will now
+assign our model to the "dev" stage as follows.
+
+![Assigning model stages](/img/mr-assign-model-stage.gif)
+
+<details id="under-the-hood-assigning-model-stages">
+
+#### 💡 Expand to see how assigning model stages works under the hood
+
+Assigning the "dev" stage to the model as we just did using DVC Studio is
+equivalent to the following GTO command
+
+```
+gto assign pool-segmentation --version v1.0.0 --stage dev
+```
+
+For more details you can have a look at the
+[gto assign command reference](doc/gto/command-reference/assign).
+
+</details>
+
+## Changing (and removing) stage assignments
+
+Let's say that we've decided to promote our model version 1.0.0 to the next
+stage. When we assing the model to the "test" stage, it will be automatically
+tested for its size (we will explore how this is done in the [Using and
+Deploying models] (doc/start/model-management/model-cicd) chapter).
+
+We will assign it to the "test" stage just like we did with the "dev" stage in
+the previous section but we also want our team to know that this model is no
+longer in the "dev" stage.
+
+To do that, we can remove the "dev" stage from the model version 1.0.0 like
+this: tested for its size (we will explore how this is done in the [Using and
+Deploying models] (doc/start/model-management/model-cicd) chapter ).
```

Comment:
Looks like this is missing something about how to actually do it? Maybe something got copied wrong here?